### DISCUSSION TITLE
Oneblock drops centered

### DISCUSSION THREAD

**Discussion by LuluxClient** [BG-Software-LLC/SSB-OneBlock#144] [Q&A] [answered]
When breaking the oneblock on your islands the items keeps flying everywhere which is normal BUT it would be cool if the items could be centered on the oneblock block just like this https://gyazo.com/8fa43cdc5754e71c5d8d3f53343e4b00 it will just be better for new players since they might lose items on the way.

**Comment by LuluxClient** [comment]
It's for the oneblock addon.

**Comment by Lemvs** [comment, marked as answer]
drop-naturally: false in OneBlock config.

@OmerBenGera you can mark this answered and close it.
